env.sh

```diff
@@ -112,4 +117,4 @@ else
    echo "Sourcing $(realpath $SHELLFILE) for environment" 
    source $SHELLFILE
    echo "$(realpath ${SHELLFILE})" > $DIR/bin/current_shell_path
-fi
+fi
```

Comment:
```suggestion
fi

```
This suggestion is to avoid unnecessary changes